setup.py

```diff
@@ -27,7 +27,8 @@
 setup(
     name="Lasagne",
     version=version,
-    description="neural network tools for Theano",
+    description="A lightweight library to build and train neural networks "
+                "in Theano",
     long_description="\n\n".join([README, CHANGES]),
     classifiers=[
         "Development Status :: 3 - Alpha",
```

Comment:
Don't know. If at all, I'd change that along with the version. We'd need to check if there are any definitions/recommendations on when to use which status.